src/components/NewTask.tsx

```diff
@@ -85,45 +100,77 @@ export const NewTask = ({ locales }: Props) => {
   const possibleLocales = locales.filter(locale => locale.enabled !== false)
 
   return (
-    <Stack space={3}>
-      <Flex align="center" justify="space-between">
-        <Text weight="semibold" size={1}>
-          {possibleLocales.length === 1 ? `Select locale` : `Select locales`}
-        </Text>
+    <Stack paddingTop={4} space={4}>
+      <Heading as="h2" weight="semibold" size={2}>
```

Comment:
```suggestion
      <Text as="h2" weight="semibold" size={2}>
```
See above comments on standards in `Heading` vs `Text`

Reply:
Sorry, I don't think the <Text> suggestion made it into the code -- would you mind applying it?